editor/code_editor.cpp

```diff
@@ -126,22 +126,18 @@ void FindReplaceBar::unhandled_input(const Ref<InputEvent> &p_event) {
 		return;
 	}
 
-	Control *focus_owner = get_viewport()->gui_get_focus_owner();
-	if (text_editor->has_focus() || (focus_owner && vbc_lineedit->is_ancestor_of(focus_owner))) {
```

Comment:
I don't think it's safe to remove this kind of check. But it seems that for some reason the check is limited to the line edit. I think we can simply check if the text editor is in focus or _any control_ within the FindReplaceBar is, not just the line edit.